test/framework/functional/base_functional_test_case.py

```diff
@@ -62,10 +49,6 @@ def _create_test_config_file(self, conf_values_to_set=None):
         return test_conf_file_path
 
     def tearDown(self):
-        # Clean up files created during this test
-        with suppress(FileNotFoundError):
-            os.remove(self.test_conf_file_path)
```

Comment:
@josephharrington It's still getting removed. At the end of `tearDown` it blows away the whole temp folder including all these files.